cmd/provisioner-localpv/app/provisioner_blockdevice.go

```diff
@@ -90,7 +93,7 @@ func (p *Provisioner) ProvisionBlockDevice(opts pvController.VolumeOptions, volu
 		WithAnnotations(volAnnotations).
 		WithReclaimPolicy(opts.PersistentVolumeReclaimPolicy).
 		WithAccessModes(pvc.Spec.AccessModes).
-		WithVolumeMode(fs).
+		WithVolumeMode(volumeMode).
 		WithCapacityQty(pvc.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]).
 		WithLocalHostPathFormat(path, fsType).
```

Comment:
Why is the `fsType` required here? 
In Raw Block mode, only the path is required in `LocalVolumeSource`.

Reply:
In case of block mode will FSType be ignored? Does path contain block path?

Reply:
Yes, path is over-ridden by the blkPath.